components/users.js

```diff
@@ -83,6 +83,27 @@ SteamCommunity.prototype.removeFriend = function(userID, callback) {
 	}, "steamcommunity");
 };
 
+SteamCommunity.prototype.getBlockedList = function (callback) {
+    this.httpRequestGet("https://steamcommunity.com/friends/blocked", function (err, res, body) {
```

Comment:
```suggestion
    this.httpRequestGet("https://steamcommunity.com/my/friends/blocked?ajax=1", function (err, res, body) {
```
Saves resources/bandwidth

Reply:
Damn didn't know that endpoint existed.
And yes it seems like the response size is ~3x smaller.